services/wallet/database.go

```diff
@@ -273,6 +273,67 @@ SELECT blocks.hash, blk_number FROM accounts_to_blocks JOIN blocks ON blk_number
 	return nil, err
 }
 
+type Token struct {
+	Address  common.Address `json:"address"`
+	Name     string         `json:"name"`
+	Symbol   string         `json:"symbol"`
+	Decimals uint           `json:"decimals"`
```

Comment:
What is `Decimal` exactly? I think the other fields are self-explanatory, but maybe worth leaving a comment describing `Decimals`